tests/mock/conftest.py

```diff
@@ -19,7 +20,9 @@ def mock_server_connection():
 @pytest.fixture(scope="function")
 def session(mock_server_connection):
     with Session(mock_server_connection) as s:
+        _add_session(s)
         yield s
+        _remove_session(s)
```

Comment:
nit: it's probably easier to just do the following -- `_add_session` is handled by `create` and `_remove_session` is handled by the ctx manager automatically for you
```suggestion
    with Session.builder.config('local_testing', True).create() as s:
        yield s
```